deps/amqp10_client/Makefile

```diff
@@ -2,13 +2,30 @@ PROJECT = amqp10_client
 PROJECT_DESCRIPTION = AMQP 1.0 client from the RabbitMQ Project
 PROJECT_MOD = amqp10_client_app
 
+define PROJECT_APP_EXTRA_KEYS
+%% Hex.pm package informations.
+	{licenses, ["MPL 2.0"]},
```

Comment:
Hex.pm now asks for an SPDX-compatible license short name. It should be `MPL-2.0` in this case.

Reply:
Good catch! Got it in the `amqp_client` `Makefile` as well.